README.md

```diff
@@ -22,6 +22,7 @@ addon | version | summary
 --- | --- | ---
 [account_payment_unece](account_payment_unece/) | 10.0.1.0.0 | UNECE nomenclature for the payment methods
 [account_tax_unece](account_tax_unece/) | 10.0.1.0.0 | UNECE nomenclature for taxes
+[base_iso3166](base_iso3166/) | 10.0.1.0.0 | ISO 3166 nomenclatures for countries
```

Comment:
No problem, and thanks anyway for your first OCA submission! It's a common mistake that maybe we can avoid if there's a way to put any text like "auto-generated" or so.